css/css-images/image-orientation/image-orientation-background-properties-border-radius.html

```diff
@@ -6,7 +6,7 @@
 <link rel="author" title="Stephen Chenney" href="mailto:schenney@chromium.org">
 <link rel="help" href="https://drafts.csswg.org/css-images-3/#propdef-image-orientation">
 <link rel="match" href="reference/image-orientation-background-properties-border-radius-ref.html">
-<meta name=fuzzy content="10;100">
+<meta name=fuzzy content="0-3;0-331">
```

Comment:
It look like this range doesn't include the original maximum difference?

Reply:
I took the maximum values from https://wpt.fyi/results/css/css-images/image-orientation/image-orientation-background-properties-border-radius.html?label=master&label=experimental&product=chrome&product=webkitgtk&product=firefox&product=safari&aligned&q=css%2Fcss-images%2Fimage-orientation%2Fimage-orientation-background-properties-border-radius.html